webpack.config.js

```diff
@@ -176,15 +176,13 @@ export default {
         type: 'asset/resource',
         generator: {
           filename: 'fonts/[name][ext]',
-          publicPath: '../', // required to remove css/ path segment
```

Comment:
Related https://github.com/webpack-contrib/mini-css-extract-plugin/releases/tag/v2.0.0. Fonts and PNG works for me without these hacks now.  🎉